server/helpers/constants.js

```diff
@@ -57,3 +57,8 @@ export const addressSchema = {
     type: String,
   },
 };
+
+export const VERIFICATION_COMMENT_STATUS = {
```

Comment:
This can be COMMENT_STATUS so that we can use it for other comment types too.

```suggestion
export const COMMENT_STATUS = {
```